src/nodeconductor_test/helpers.py

```diff
@@ -260,15 +318,55 @@ def create_provider(driver, provider_name):
     provider_creation = driver.find_element_by_link_text('Create provider')
     provider_creation.click()
     time.sleep(5)
-    provider_type = driver.find_element_by_class_name('digitalocean')
+    provider_type = driver.find_element_by_class_name(provider_type_name)
     provider_type.click()
     time.sleep(5)
     provider_name_field = driver.find_element_by_css_selector('[ng-model="ServiceAdd.model.serviceName"]')
     provider_name_field.clear()
     provider_name_field.send_keys(provider_name)
     token_name_field = driver.find_element_by_id('DigitalOcean_token')
-    token_name_field.send_keys('6ac9ad515e61dc80fddd6f9ee83f3b866fa2b6dc8cc1274dd2becc89241dd710')
-    # ('dd7b13dabd7a3579885bba9de6482da15f0a5305dd2c22afc13eadf3e04c8ffe')
+    token_name_field.send_keys(token_name)
+    add_provider_button = driver.find_element_by_link_text('Add provider')
+    add_provider_button.click()
+
+
+def create_provider_azure(driver, provider_name, provider_type_name, subscription_id_name):
+    organization_field = driver.find_element_by_css_selector('ul.nav-list span.customer-name')
+    organization_field.click()
+    time.sleep(5)
+    organization_details = driver.find_element_by_link_text('Details')
+    organization_details.click()
+    time.sleep(10)
+    providers = driver.find_element_by_css_selector('[visible="providers"]')
+    providers.click()
+    time.sleep(5)
+    provider_creation = driver.find_element_by_link_text('Create provider')
+    provider_creation.click()
+    time.sleep(5)
+    provider_type = driver.find_element_by_class_name(provider_type_name)
+    provider_type.click()
+    time.sleep(5)
+    provider_name_field = driver.find_element_by_css_selector('[ng-model="ServiceAdd.model.serviceName"]')
+    provider_name_field.clear()
+    provider_name_field.send_keys(provider_name)
+    token_name_field = driver.find_element_by_id('Azure_username')
+    token_name_field.send_keys(subscription_id_name)
+    upload_file = driver.find_element_by_link_text('Browse')
+    upload_file.click()
+    time.sleep(5)
+    # certificate = driver.switch_to_window('File Upload')
+    certificate = driver.switch_to_active_element()
+    certificate.send_keys('home\julia\azure_cert.pem')
+    certificate.submit()
+    # certificate = driver.find_element_by_name('azure_cert.pem')
+    # certificate.open()
+    # driver.current_window_handle
+    # element = driver.switch_to_window("File Upload")
+    # driver.navigate()
+    # element.send_keys("home\julia\azure_cert.pem")
+    # alert = driver.switch_to_alert()
+    # alert.accept()
```

Comment:
As I remember certificate addition code is not working. Please remove it from file and add some # TODO comment to `create_provider_azure` method